### ISSUE TITLE
Bug report form

### ISSUE THREAD

**Issue by twjeffery** [GovAlta/ui-components-design#45]
_As a member of a product team building a service, I want to report a bug easily to the design system and give all required information_

In order to better support the support requests we get on the design system, we had the idea to create a bug request form on our design system website. This would allow members of a product team to fill out bug information and automatically create an issue in github, without having to log out and in of their Github account to submit an issue to us. This form would also contain the necessary pieces of mandatory information we need to understand and respond to the request.

**Fields from Dustin:**
- email address
- Web Components version
- React/Angular Components version
- Description of bug
- Replication steps
- Expected Output
- Actual Output
- Code to reproduce bug
- Screenshots and/or media (video link)
- Any Additional Info

**Comment by twjeffery** [comment]
@Spark450 I created an issue here to track this work. 

I put a design together here to review: 
https://www.figma.com/design/qWGWDvULAAZuVaDpXCRzqP/DS-Reference-website?node-id=14678-294461&t=ohRi56spfG83vucj-4

I tried to incorporate the form into the existing content we have. I thought it was useful to differentiate the request at the start between bug report and feature request, as that is the main issue types we currently have in Github. For the feature request, I linked to and referenced the recent update we are making to the contribution process.

<img width="862" alt="image" src="https://github.com/user-attachments/assets/4d55af12-5654-4855-9709-b3e55a9215a5">


Let me know if you have any questions, happy to go through it together.
